inference-engine/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp

```diff
@@ -335,19 +336,30 @@ void LayerTestsCommon::LoadNetwork() {
     executableNetwork = core->LoadNetwork(cnnNetwork, targetDevice, configuration);
 }
 
+void LayerTestsCommon::GenerateInputs() {
```

Comment:
General comment, I don't see the main reason of these changes. Looks like you have tried to simplify `Infer()` method using additional memory.  Is it correct?

Reply:
Aim: generation of the inputs is separated from infer and could be used without infer.
No change to the memory usage is caused by this decomposition.